packages/data-service-generator/src/server/package-json/package.json

```diff
@@ -64,7 +64,7 @@
     "prisma": "^5.4.2",
     "supertest": "^6.3.3",
     "ts-jest": "^29.1.1",
-    "typescript": "^5.2.2"
```

Comment:
This would potentially bring breaking changes to our users generated code as it potentially force them to an earlier version of TS.

The real issue is related to the types mapping we generate that are not

Reply:
@overbit we tested it and it works both on run and build.
The problematic version is `5.4.2` and the chance that some user has custom code that relies on that version is very minor.